pkg/kubelet/kubelet_getters.go

```diff
@@ -237,12 +237,20 @@ func (kl *Kubelet) GetNode() (*v1.Node, error) {
 // in which case return a manufactured nodeInfo representing a node with no pods,
 // zero capacity, and the default labels.
 func (kl *Kubelet) getNodeAnyWay() (*v1.Node, error) {
```

Comment:
this is used in several places. do all callers react well to switching this behavior?